sycl/doc/extensions/USMAddressSpaces/usm_address_spaces.asciidoc

```diff
@@ -1,4 +1,4 @@
-= SYCL_INTEL_usm_address_spaces
+= SYCL_EXT_INTEL_USM_ADDRESS_SPACES
```

Comment:
Should "Name Strings" section be updated as well?

Reply:
Fixed in https://github.com/intel/llvm/pull/4432/commits/8a83962d83b4e3d4bb38de25550112b7b497f210